### PR TITLE
[RASDLG] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 1]
```diff
@@ -76,8 +76,8 @@ BEGIN
     AUTOCHECKBOX "Micr&osoft CHAP versiunea 2 (MS-CHAP v2)", 1492, 28, 176, 200, 10, BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 28, 189, 208, 1
     AUTOCHECKBOX "Pentru protocoalele &bazate pe MS-CHAP, utilizează nume și parole de autentificare în ReactOS (și eventual, domenii)", 1549, 28, 195, 213, 24, BS_TOP | BS_MULTILINE
-    DEFPUSHBUTTON "Con&firmă", 1, 127, 232, 60, 14
-    PUSHBUTTON "A&nulează", 2, 192, 232, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 127, 232, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:79]: This change introduces an accelerator collision with the line
 AUTOCHECKBOX "Micr&osoft CHAP versiunea 2 (MS-CHAP v2)", 1492, 28, 176, 200, 10, BS_TOP | BS_MULTILINE
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 2]
```diff
@@ -76,8 +76,8 @@ BEGIN
...
+    PUSHBUTTON "&Revocare", 2, 192, 232, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:80]: This change introduces an accelerator collision with the line
  PUSHBUTTON "P&roprietăți", 1035, 184, 77, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 3]
```diff
@@ -106,50 +106,50 @@ BEGIN
     AUTOCHECKBOX "&Aplică regulile de formare a numerelor", 1048, 20, 81, 219, 10, BS_TOP | BS_MULTILINE
     GROUPBOX "Co&mentariu", 1052, 7, 102, 243, 36
     EDITTEXT 1050, 17, 115, 224, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 127, 147, 60, 14
-    PUSHBUTTON "A&nulează", 2, 191, 147, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 127, 147, 60, 14
+    PUSHBUTTON "&Revocare", 2, 191, 147, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:110]: This change introduces an accelerator collision with the line
 LTEXT "Codul &regiunii:", 1055, 18, 17, 54, 8
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 4]
```diff
@@ -106,50 +106,50 @@ BEGIN
...
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Aveți aleasă opțiunea numărului specificat de către apelat. Apăsați Confirmă pentru a fi apelat de către server la numărul de mai jos. Apăsați Anulează pentru a omite retroapelarea.", 1066, 5, 5, 221, 39, SS_NOPREFIX
+    LTEXT "Aveți aleasă opțiunea numărului specificat de către apelat. Apăsați OK pentru a fi apelat de către server la numărul de mai jos. Apăsați Revocare pentru a omite retroapelarea.", 1066, 5, 5, 221, 39, SS_NOPREFIX
     LTEXT "Int&roduceți numărul de telefon al modemului dumneavoastră:", 1067, 5, 46, 222, 8, NOT WS_GROUP
     EDITTEXT 1065, 5, 57, 221, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 100, 85, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 164, 85, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 100, 85, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 164, 85, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:152]: This change introduces an accelerator collision with the line
 LTEXT "Int&roduceți numărul de telefon al modemului dumneavoastră:", 1067, 5, 46, 222, 8, NOT WS_GROUP
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 5]
```diff
@@ -213,14 +213,14 @@ CAPTION "Conectare %1"
...
     COMBOBOX 1416, 88, 56, 165, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
-    LTEXT "Apelează &din:", 1415, 9, 77, 71, 8
+    LTEXT "Apelare &din:", 1415, 9, 77, 71, 8
     COMBOBOX 1414, 88, 75, 89, 112, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | NOT WS_TABSTOP
     PUSHBUTTON "&Reguli de apel", 1108, 181, 75, 72, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 8, 96, 245, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 8, 103, 61, 14
-    PUSHBUTTON "A&nulează", 1591, 70, 103, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 8, 103, 61, 14
+    PUSHBUTTON "&Revocare", 1591, 70, 103, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:223]: This change introduces an accelerator collision with the line
 PUSHBUTTON "&Reguli de apel", 1108, 181, 75, 72, 14
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 6]
```diff
@@ -259,12 +259,12 @@ BEGIN
...
     EDITTEXT 1102, 102, 95, 154, 14, ES_UPPERCASE | ES_AUTOHSCROLL
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN, 9, 116, 247, 1
-    AUTOCHECKBOX "Păstrea&ză acest nume și parolă pentru acești utilizatori:", 1101, 12, 123, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
+    AUTOCHECKBOX "Sal&vare a acestor nume și parolă pentru acești utilizatori:", 1101, 12, 123, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 25, 137, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 25, 151, 230, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 9, 167, 247, 1
     DEFPUSHBUTTON "&Apelează", 1590, 8, 173, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 71, 173, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 71, 173, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:267]: This change introduces an accelerator collision with the line below
 PUSHBUTTON "P&roprietăți", 1107, 133, 173, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 7]
```diff
@@ -282,15 +282,15 @@ BEGIN
...
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 29, 139, 225, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 29, 152, 225, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 167, 247, 1
     LTEXT "Apelea&ză:", 1412, 10, 177, 90, 8
     COMBOBOX 1416, 102, 173, 154, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 192, 247, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 10, 199, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 72, 199, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 10, 199, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 1591, 72, 199, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:293]: This change introduces an accelerator collision with the line below
PUSHBUTTON "P&roprietăți", 1107, 134, 199, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 8]
```diff
@@ -306,15 +306,15 @@ BEGIN
...
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 24, 117, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 24, 132, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 147, 247, 1
     LTEXT "Ap&elează:", 1412, 13, 157, 88, 8
     COMBOBOX 1416, 103, 154, 154, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 173, 247, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 10, 180, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 72, 180, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 10, 180, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 1591, 72, 180, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:317]: This change introduces an accelerator collision with the line below
PUSHBUTTON "P&roprietăți", 1107, 134, 180, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 9]
```diff
@@ -457,8 +457,8 @@ BEGIN
     EDITTEXT 1404, 19, 120, 193, 14, ES_AUTOHSCROLL
     LTEXT "F&acilități:", 1409, 19, 138, 182, 8, NOT WS_GROUP
     EDITTEXT 1403, 19, 149, 193, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 100, 189, 60, 14
-    PUSHBUTTON "A&nulează", 2, 164, 189, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 100, 189, 60, 14
+    PUSHBUTTON "&Revocare", 2, 164, 189, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:461]: This change introduces an accelerator collision with the line
 LTEXT "&Rețea:", 1410, 9, 35, 213, 8, NOT WS_GROUP
Please check your romanian Windows to find out which of both lines need to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 10]
```diff
@@ -1046,7 +1046,7 @@ BEGIN
     PUSHBUTTON "&Reguli de apel", 1108, 193, 110, 64, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 7, 130, 250, 1
     DEFPUSHBUTTON "&Apelează", 1590, 6, 136, 60, 14
-    PUSHBUTTON "A&nulează", 1591, 69, 136, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 69, 136, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:1049]: This change does introduce an accelerator collision with the line
PUSHBUTTON "&Reguli de apel", 1108, 193, 110, 64, 14
Check your Romanian Windows to find out which one needs to be changed.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 11]
```diff
@@ -1451,7 +1451,7 @@ BEGIN
     PUSHBUTTON "Reguli de apel", 1108, 193, 195, 65, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 215, 248, 1
     DEFPUSHBUTTON "&Apelează", 1590, 8, 221, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 71, 221, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 71, 221, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:1454]: Accelerator collision vs.
PUSHBUTTON "P&roprietăți", 1107, 134, 221, 60, 14
Check your Romanian Windows to find out which to change.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 12]
```diff
@@ -1701,8 +1701,8 @@ BEGIN
     LTEXT "Total:", -1, 29, 124, 53, 10
     LTEXT "", 7024, 90, 121, 150, 14, 0, WS_EX_STATICEDGE
     AUTOCHECKBOX "Pe&rmite calculatorului apelator specificarea unei adrese IP proprii", 7022, 20, 144, 231, 10, BS_TOP | BS_MULTILINE | WS_GROUP
-    DEFPUSHBUTTON "Con&firmă", 1, 133, 169, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 197, 169, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 133, 169, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 197, 169, 60, 14
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:1705]: Accelerator collision &R vs.
 AUTOCHECKBOX "Pe&rmite calculatorului apelator specificarea unei adrese IP proprii", 7022, 20, 144, 231, 10, BS_TOP | BS_MULTILINE | WS_GROUP
Check your Romanian Windows to find out which to change.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Do not merge that PR, it breaks much more regarding the accelerators than it improves things with the translations.

**Comment by AndreiMiloiu** [comment]
I fixed them! I hope it's good now.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 13]
```diff
@@ -2326,7 +2326,7 @@ BEGIN
     7351 "Sistemul nu poate șterge utilizatorul cerut.\0"
     7352 "Componenta selectată de rețea nu poate fi eliminată, deoarece încă este în uz.\0"
     7353 "Avertisment al conexiunilor de intrare\0"
-    7354 "Un utilizator șters va fi eliminat definitiv din sistem, chiar dacă va fi apăsat Anulează. Sigur doriți eliminarea irevocabilă a utilizatorului %s? \0"
+    7354 "Un utilizator șters va fi eliminat definitiv din sistem, chiar dacă va fi apăsat Revocare. Sigur doriți eliminarea irevocabilă a utilizatorului %s? \0"
```
Comment (JoachimHenze) [inline dll/win32/rasdlg/lang/ro-RO.rc:2329]:
```suggestion
    7354 "Un utilizator șters va fi eliminat definitiv din sistem, chiar dacă va fi apăsat Revocare. Sigur doriți eliminarea irevocabilă a utilizatorului %s?\0"
```

**Comment by AndreiMiloiu** [comment]
I'm ready. I corrected every mistake I discovered. Now everything is like in MSDN, except the three dot which are only on character long. However, I don't understand why do I have errors? However, at 102, 155, 554 the text cannot fit in the buttons. Shouldn't it be fixed by a new PR? I can open a ticket for it. The problem persists in XP, as I attached some screnshots.

2024-03-09 23_17_43-Resource Hacker - rasdlg dll mui

2024-03-09 23_17_58-Resource Hacker - rasdlg dll mui

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 14]
```diff
@@ -1826,267 +1826,267 @@ END
...
+    190 "Se apelează numai primul dispozitiv disponibil\0"
+    191 "Se apelează toate dispozitivele\0"
+    192 "Se apelează numai dispozitivele necesare\0"
+    193 "Se introduce un nume de utilizator şi parola de acces la domeniul reţelei de la distanţă.\0"
+    194 "Network Address Translation (NAT) (Traducere adresă reţea) este instalat curent ca protocol de distribuire,\nşi trebuie eliminat înainte de activarea Partajării conexiunii la Internet.\nPentru a elimina Network Address Translation, deschideţi Manager acces la distanţă şi distribuire\nşi extindeţi intrarea distribuitorului în panoul din stânga.\nŞtergeţi protocolul de distribuire Network Address Translation din lista protocoalelor IP\nde distribuire.\0"
+    195 "Codul zonal trebuie să conţină numai cifre de la 0 la 9.\0"
+    196 "Numele intrării trebuie să conţină cel puţin un caracter şi nu trebuie să înceapă cu un punct. Alegeţi alt nume.\0"
+    197 "Pentru acest câmp trebuie să alegeţi o valoare de la %1 la %2.\0"
+    198 "Imposibil de încărcat DLL-ul de configurare pentru autorizare particularizată. Asiguraţi-vă că pachetul de autorizare particularizată este corect instalat.\00"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:1849]:
```suggestion
    198 "Imposibil de încărcat DLL-ul de configurare pentru autorizare particularizată. Asiguraţi-vă că pachetul de autorizare particularizată este corect instalat.\0"
```

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 15]
```diff
@@ -1826,267 +1826,267 @@ END
...
-    206 "Doriți deconectarea de la %1?\0"
-    207 "Deja există un fișier numit %1. Suprascrieți?\0"
-    208 "Au fost depistate erori de sintaxă în script. Doriți un raport al erorilor?\0"
-    209 "Conectarea la '%1' a fost realizată.\0"
-    210 "Pentru a închide sau a verifica starea conexiunii, faceți clic pe Monitorul de rețea al liniei comutate din bara de activități.\0"
-    212 "Conectarea a fost realizată.\0"
-    213 "Procedura de autentificare particularizată trebuie pornită prin apăsarea butonului Proprietăți.\0"
+    201 "Configurarea pachetului de autorizare particularizată nu a reuşit.\0"
+    202 "TCP/IP neinstalat sau dezactivat pentru Distribuire şi acces la distanţă. Din Panoul de control, lansaţi Reţea, salt la pagina de Servicii, selectaţi Proprietăţi pentru 'Serviciul de distribuire şi acces la distanţă' şi apăsaţi pe butonul Reţea.\0"
+    203 "IPX neinstalat sau dezactivat pentru Distribuire şi acces la distanţă. Instalaţi/activaţi IPX vizualizând proprietăţile fiecărei conexiuni din folderul \"Conexiuni\".\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:1854]:
```suggestion
    203 "IPX neinstalat sau dezactivat pentru Distribuire şi acces la distanţă. Instalaţi/activaţi IPX vizualizând proprietăţile fiecărei conexiuni din folderul ""Conexiuni"".\0"
```

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 16]
```diff
@@ -1826,267 +1826,267 @@ END
...
+    193 "Se introduce un nume de utilizator şi parola de acces la domeniul reţelei de la distanţă.\0"
+    194 "Network Address Translation (NAT) (Traducere adresă reţea) este instalat curent ca protocol de distribuire,\nşi trebuie eliminat înainte de activarea Partajării conexiunii la Internet.\nPentru a elimina Network Address Translation, deschideţi Manager acces la distanţă şi distribuire\nşi extindeţi intrarea distribuitorului în panoul din stânga.\nŞtergeţi protocolul de distribuire Network Address Translation din lista protocoalelor IP\nde distribuire.\0"
+    195 "Codul zonal trebuie să conţină numai cifre de la 0 la 9.\0"
+    196 "Numele intrării trebuie să conţină cel puţin un caracter şi nu trebuie să înceapă cu un punct. Alegeţi alt nume.\0"
+    197 "Pentru acest câmp trebuie să alegeţi o valoare de la %1 la %2.\0"
+    198 "Imposibil de încărcat DLL-ul de configurare pentru autorizare particularizată. Asiguraţi-vă că pachetul de autorizare particularizată este corect instalat.\00"
+    199 "Conexiunea selectată a fost stabilită de distribuitorul multi-protocol.\nUtilizaţi instrumentul 'Administrare distribuire şi RAS' pentru a suspenda conexiunea.\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:1850]:
```suggestion
    199 "Conexiunea selectată a fost stabilită de distribuitorul multi-protocol.\nUtilizaţi instrumentul ""Administrare distribuire şi RAS"" pentru a suspenda conexiunea.\0"
```

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 17]
```diff
@@ -1826,267 +1826,267 @@ END
...
+    366 "Prefixe număr de telefon\0"
+    367 "Unul sau mai multe protocoale de reţea solicitate nu au reuşit să se conecteze.\n\n\0"
+    368 "Apăsaţi pe Acceptare pentru a utiliza conexiunea aşa cum este sau pe Suspendare pentru deconectare.\0"
+    369 "Setări pentru protocolul punct-la-punct, de utilizat când %1 iniţiază o conexiune de apelare la cerere pe interfaţa %2:\0"
+    370 "[Pentru a modifica parola salvată, clic aici]\0"
+    405 "Protocolul NetBEUI este obligatoriu pentru apelarea serverelor RAS mai vechi.\0"
+    406 "Înainte de a apela servere RAS mai vechi trebuie să instalaţi protocolul NetBEUI. NetBEUI se poate instala cu miniaplicaţia Reţea din Panoul de control.\0"
+    407 "Fo&rmare număr din nou\0"
+    408 "Aţi ales să dezactivaţi unul sau mai multe transporturi. Pentru aceasta se cere ca managerii de distribuire şi protocoalele de distribuire pentru transporturile dezactivate să fie eliminate din interfaţa de apelare la cerere.\nFaceţi clic pe Da dacă vreţi să continuaţi\n \n\n\n\n\n\nsau pe Nu dacă vreţi să reactivaţi transporturile.\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:2024]: Please double check, something odd here with "\n\n\n\n ..."

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 18]
```diff
@@ -1826,267 +1826,267 @@ END
...
+    409 "Partajare conexiune la Internet\0"
+    410 "Cont de apelare dinspre exterior pentru interfaţă de apelare la cerere.\0"
+    411 "Este necesar să stabiliţi acreditările pentru apeluri dinspre interior pe care le va utiliza interfaţa '%1' când se conectează la distribuitorul la distanţă. Aceste acreditări trebuie să se potrivească cu acreditările pentru apeluri dinspre exterior configurate pe distribuitorul la distanţă. Când vă conectaţi la un distribuitor \n \n\n\n\n\n\nWindows NT4/2000, numele utilizatorului trebuie de asemenea să se potrivească cu numele interfeţei distribuitorului la distanţă.\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:2027]: Same here.

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 19]
```diff
@@ -2103,279 +2103,279 @@ BEGIN
...
+    576 "20 de minute\0"
+    579 "Specificaţi un nume pentru această conexiune la locul de lucru.\0"
+    581 "Setări aplicaţie în reţea\0"
+    582 "Tastaţi un nume pentru această conexiune în caseta următoare.\0"
+    583 "Tastaţi numele celuilalt computer în caseta următoare.\0"
+    584 "Tastaţi numele furnizorului ISP în caseta următoare.\0"
+    585 "Care este numele serviciului care vă furnizează conexiunea la Internet?\0"
+    586 "Tastaţi adresa IP a computerului care este server în reţeaua privată.\0"
+    587 "Setări complexe\0"
+    588 "Numele de intrare începe cu un punct sau conţine caractere incorecte. \nAlegeţi alt nume.\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:2170]:
```suggestion
    588 "Numele de intrare începe cu un punct sau conţine caractere incorecte.\nAlegeţi alt nume.\0"
```

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 20]
```diff
@@ -2103,279 +2103,279 @@ BEGIN
...
-    1646 "Acceptă cererea de ecou\0"
-    1647 "Acceptă cererea de marcaj temporal\0"
-    1648 "Acceptă cererea de mască\0"
-    1649 "Acceptă cererea de redirecționare\0"
-    1650 "Permite răspunsul de destinație inaccesibilă\0"
-    1651 "Permite răspunsul de înăbușire a sursei\0"
-    1652 "Permite răspunsul problemă de parametru\0"
-    1653 "Permite răspunsul depășirii limitei de timp\0"
-    1654 "Permite redirecționarea\0"
+    1612 "Aţi selectat utilizarea unei chei pre-distribuite dar nu aţi introdus nici una. \nIntroduceţi cheia pre-distribuită.\0"
```
Comment (binarymaster) [inline dll/win32/rasdlg/lang/ro-RO.rc:2244]:
```suggestion
    1612 "Aţi selectat utilizarea unei chei pre-distribuite dar nu aţi introdus nici una.\nIntroduceţi cheia pre-distribuită.\0"
```

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**